job-dsls/jobs/kieAll_meta_pipeline.groovy

```diff
@@ -36,7 +36,7 @@ def sendUMB="""pipeline{
                     "7.48.x-pipeline" : {
                         build job: "../7.48.x/daily-build/daily-build-pipeline-7.48.x", propagate: false
                     },
-                    "master-jdk11-pipeline" : {
+                    "7.48.x-jdk11-pipeline" : {
```

Comment:
i think we can skip 7.48.x branch for jdk11 to work as we know all the work now is only on master branch